README.md

```diff
@@ -338,10 +338,10 @@ secrets:
 
 We don't support installing plugins with JCasC you need to use something else for this,
 
-Dockers users can use:  
+Dockers users can use:
```

Comment:
On markdown please do no trim double white spaces. Double white spaces and newline creates a line break
https://gist.github.com/shaunlebron/746476e6e7a4d698b373
```suggestion
Dockers users can use:  
```